components/job-orchestration/pyproject.toml

```diff
@@ -16,6 +16,7 @@ mysql-connector-python = "^8.2.0"
 pika = "^1.3.2"
 pydantic = "^1.10.13"
 PyYAML = "^6.0.1"
+StrEnum = "^0.4.15"
```

Comment:
Don't think this is a direct dependency of job-orchestration, right?

Reply:
You're right, I'll remove it